patterns/2-structured/agile-innersource-dojo.md

```diff
@@ -0,0 +1,76 @@
+
+## Forces
+
+* **Lack of Awareness**: Many Agile practitioners are unaware of InnerSource methodologies and vice versa.
+* **Diversity of Participants**: The Dojo involves individuals with different backgrounds, expertise and expectations.
+* **Need for Safe Spaces**: Participants must feel safe to share their insights and challenges.
+* **Continuous Engagement**: Ongoing participation is crucial for maintaining the Dojo's momentum and impact.
+* **Knowledge Sharing**: Effective methods for sharing knowledge and best practices are essential.
+
+## Solutions
+
+Adopt a strategic approach to create and sustain an Agile InnerSource Dojo, focusing on clear objectives, inclusive participation and continuous improvement.  This can be achieved through the following parallel efforts:
+
+* **Define the Purpose**: Clearly articulate the Dojo's purpose and goals, ensuring alignment with broader organizational objectives.
+* **Establish Social Conventions**: Develop and communicate social conventions that promote respect, inclusivity and open dialogue.  Encourage members to embrace and evolve these guidelines to create a safe and supportive environment.
+* **Facilitate Regular Meetings**: Schedule regular meetings to maintain engagement and momentum.  Use these sessions for open discussions, sharing insights and addressing challenges.  Meetings are open for unpacking issues, with goals set by attendees.
+* **Rotate Leadership**: Encourage shared leadership by rotating the facilitation role among members.  This helps distribute responsibility and fosters a sense of connection.
+* **Use Collaborative Tools**: Utilize tools like Jitsi Meet, Slack and GitHub for communication and documentation.  Insights can be optionally shared on social channels, ensuring privacy.
+* **Encourage Voluntary Code Contribution**: Foster an environment where contributions are voluntary, allowing individuals to work on what interests them and aligning with their passions.
+* **Prioritized Mentorship**: Implement a mentorship structure with Sensei (teachers) providing guidance and Senpai (senior students) supporting newer members. This dual-layer mentorship enhances skill development and community cohesion.
+* **Foster a Culture of Continuous Improvement**: Regularly seek feedback to identify improvement areas, refining the Dojo's conditioning and activities accordingly.
+
+## Resulting Context
+
+Embracing this pattern helps InnerSource programs become vibrant, effective and sustainable.  Participants will feel more connected and engaged, leading to increased knowledge sharing, innovation and collective growth.
+
+## Related Patterns
+
+* [Dedicated Community Leader](https://patterns.innersourcecommons.org/p/dedicated-community-leader)
```

Comment:
```suggestion
* [Dedicated Community Leader](../2-structured/dedicated-community-leader.md)
```

Better to use relative links to the markdown files of the other patterns. These will be turned into correct links in our book automatically.

When linking to files in folder 1-initial, these will then link to the files in our github repo, rather than in our online book.

So in other words, everything will work as intended out of the box once you use relative links here :)

Could you also change this for the others?